parkings/tests/api/operator/test_disc_parking.py

```diff
@@ -64,5 +67,6 @@ def test_delete_disc_parking(operator_api_client, disc_parking):
 
 
 def test_required_fields_for_disc_parking(operator_api_client, operator, disc_parking_data):
+    create_enforced_domain_for_user(operator.user)
```

Comment:
Same here. Is this needed? https://github.com/City-of-Helsinki/parkkihubi/pull/146/commits/d2cd37559501da7fbec0130bfe7d4f9ad860b437#r412981973